mod_reforged/config/items.nut

```diff
@@ -0,0 +1,27 @@
+::Reforged.Items <- {
+	function isDuelistValid( _weapon )
+	{
+		if (_weapon.isEquipped())
+			return _weapon.getContainer().getActor().getSkills().getAttackOfOpportunity().isDuelistValid();
```

Comment:
Where are you checking whether the weapon has an AOO skill with 4 AP or lower like you states in the branch description?

I see you removed the old condition from inside the skill.nut hook but as far as I see now there is no more AP check at all and every melee weapon with a 1 range is duelist viable.